pkg/reconciler/managed/api_test.go

```diff
@@ -411,6 +424,10 @@ func TestRetryingCriticalAnnotationUpdater(t *testing.T) {
 			if diff := cmp.Diff(tc.want, got, test.EquateErrors()); diff != "" {
 				t.Errorf("\n%s\nu.UpdateCriticalAnnotations(...): -want, +got:\n%s", tc.reason, diff)
 			}
+			getCalled := tc.args.o.GetAnnotations()["getcalled"] == "true"
+			if getCalled != tc.wantGetCalled {
+				t.Errorf("\n%s\nu.UpdateCriticalAnnotations(...) calling get: -want %v, +got %v", tc.reason, tc.wantGetCalled, getCalled)
+			}
```

Comment:
```go
			// tc.want.o replaces tc.wantGetCalled
			if diff := cmp.Diff(tc.want.o, tc.args.o,); diff != "" {
				t.Errorf("\n%s\nu.UpdateCriticalAnnotations(...): -want, +got:\n%s", tc.reason, diff)
			}
```

Nit: Typically in this case where we're testing for a mutation of the supplied object we'd use `cmp.Diff` on the whole object as above. A little overkill here, but more extensible if we ever want to test more things about the object we're mutating.